sys/aarch64/timer.c

```diff
@@ -0,0 +1,98 @@
+#define KL_LOG KL_TIME
+#include <sys/klog.h>
+#include <sys/timer.h>
+#include <aarch64/armreg.h>
+#include <aarch64/bcm2835reg.h>
+#include <sys/interrupt.h>
+#include <sys/bus.h>
+#include <sys/devclass.h>
+
+#define CNTCTL_ENABLE 1
+
+typedef struct arm_timer_state {
+  intr_handler_t intr_handler;
+  timer_t timer;
+  uint64_t step;
+} arm_timer_state_t;
+
+static int arm_timer_start(timer_t *tm, unsigned flags, const bintime_t start,
```

Comment:
```flags``` and ```start``` are unused. They are required by definition of ```tm_start_t```, but maybe mark them as ```__unused```?

Reply:
done